layouts/_default/baseof.html

```diff
@@ -55,10 +55,12 @@
 </head>
 
 <body id="{{ block "bodyID" . }}home{{ end }}">
+
     <main class="page-content" aria-label="Content">
     {{ block "main" . }}{{ end }}
     </main>
 
+    {{ partial "header.html" }}
```

Comment:
Semantically, do you think this should be above `<main>`? I'm not a huge stickler - just asking.

Reply:
I assumed it was last for REASONS and left it down there. (Maybe to do with it being sticky?) I can move it up and test though.